src/dymaptic.GeoBlazor.Core/Components/Widgets/BasemapLayerList/BaseListItem.cs

```diff
@@ -0,0 +1,33 @@
+﻿using dymaptic.GeoBlazor.Core.Components.Widgets.LayerList;
+using Microsoft.AspNetCore.Components;
+
+
+namespace dymaptic.GeoBlazor.Core.Components.Widgets.BasemapLayerList;
+/// <summary>
+///     The ListItem class represents one of the operationalItems in the LayerListViewModel. In the LayerList widget UI, the list item represents a layer displayed in the view. It provides access to the associated layer's properties, allows the developer to configure actions related to the layer, and allows the developer to add content to the item related to the layer.
+///     <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-BasemapLayerList.html">ArcGIS JS API</a>
+/// </summary>
+public class BaseListItem : ListItem
```

Comment:
This entire class is unnecessary. In the JS, `BasemapLayerList.baseItems` and `BasemapLayerList.referenceItems` are both collections of the type `ListItem`. Using this class instead causes the following problems:
1. You are overriding `ActionSections` without an `override` or `new` keyword, which should be throwing a warning.
2. You now have three child collections, `Children`, `BaseItems`, and `ReferenceItems`
3. You could in theory add a `ReferenceItem` as a child of a `BaseItem`, or vice-versa.

Also, it's probably on one of my open PRs, but since `ListItem` is not a razor component, the `[Parameter]` attribute is meaningless.

Reply:
removed and only reference list item